hypertrace-view-generator/hypertrace-view-generator/src/test/java/org/hypertrace/viewgenerator/generators/BackendEntityViewGeneratorTest.java

```diff
@@ -0,0 +1,45 @@
+package org.hypertrace.viewgenerator.generators;
+
+import org.apache.avro.file.DataFileReader;
+import org.apache.avro.specific.SpecificDatumReader;
+import org.hypertrace.core.datamodel.Event;
+import org.hypertrace.core.datamodel.StructuredTrace;
+import org.hypertrace.viewgenerator.api.BackendEntityView;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class BackendEntityViewGeneratorTest {
+
+    @Test
+    public void testBackendEntityViewGenerator_HotrodTrace() throws IOException {
+        URL resource = Thread.currentThread().getContextClassLoader().getResource("StructuredTrace-Hotrod.avro");
```

Comment:
should we consider using JSON format in the test instead of avro?
e.g https://www.baeldung.com/java-apache-avro
cc : @rish691

Reply:
Yes. Otherwise, it's not easy to fix unit tests later on. We already have some unit tests on the same line in other view generators I believe

Reply:
@Sunn-y-Arora I saw your slack request of addressing as follow-up PR. yes, we can do that as well. But, would prefer that we start with one test where we start using some readable format (like JSON) for testing. As, the current, way is hard to understand.